### DISCUSSION TITLE
如何提高分辨率

### DISCUSSION THREAD

**Discussion by IR141** [422658476/MPV-EASY-Player#71] [Q&A【问答】]
使用mpv.net时发现清晰度比mpv-easy高很多，请问有什么办法可以提高mpv-easy的清晰度

**Comment by 422658476** [comment]
@IR141 

这很有可能是因为mpv.net没有使用mpv的默认图像缩放算法导致的

如果你在mpv-easy player 的【data\mpv-easy-data\rjno1.conf】的[ini-rjno1]中填入下面2行或者其中1行，那么应该会获得相同或者类似的效果。

scale=oversample
dscale=oversample

注意： scale的值会在视频画面放大时激活，dscale则是在视频画面缩小时激活，通常应该先测试dscale，因为通常显示器分辨率或者视频窗口尺寸比视频分辨率低，视频会缩小后播放。

scale=和dscale=后能填哪些值，以及这些设置的意义，请查阅

https://mpv.io/manual/stable/#options-scale
https://mpv.io/manual/stable/#options-dscale

请注意，这些使画面“锐利”的设置会导致边缘出现锯齿，也就是过渡不平滑

如果你不想要使用这种方式使图像更加锐利，也可以尝试使用：

--sharpen=3    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;#----> 2-4可能是一个较好的值

使用前也请参考

https://mpv.io/manual/stable/#options-sharpen

如果你想深入的研究哪种图像缩放算法更加合适，可以参考这篇文章：

https://artoriuz.github.io/blog/mpv_upscaling.html

目前（2023-08-05）mpv默认的dscale的算法为michell（ 2023-11-22 mpv 0.37.0开始默认dscale算法改为hermite），这篇文章的作者在文章底部给出了三种不同的推荐算法（2023-08-05）为：mitchell 或者 catrom 或者 lanczos

mpv目前使用什么算法，请按下【`】，开启命令控制台，输入：

show-text ${dscale}

即可在窗口右上角看到结果
